docker/slackbolt/Dockerfile

```
@@ -0,0 +1,8 @@
+
+FROM demisto/python3:3.10.13.86272
```

In order to use the demisto-sdk you please change the base image to 
```suggestion
FROM demisto/demisto-sdk:1.26.2.87554
```
Which already includes demisto-sdk